.github/workflows/release.yaml

```diff
@@ -17,10 +17,10 @@ jobs:
             PYINSTALLER_ARGS: "--onefile"
           - os: macos-latest
             APP_NAME: "USDB Syncer.app"
```

Comment:
APP_NAME should be "USDB Syncer", right? The `.app` is appended as part of the `pyinstaller --windowed` process, I believe.